lib/cucumber/formatter/message_builder.rb

```diff
@@ -42,11 +42,12 @@ def output_message
         raise 'To be implemented'
       end
 
-      def attach(src, media_type)
+      def attach(src, media_type, filename = 'jeff')
```

Comment:
Note to self, remove this when we're happy that it's passing through all the consumers